pyvista/core/errors.py

```diff
@@ -46,3 +46,13 @@ def __init__(
     ):
         """Empty init."""
         RuntimeError.__init__(self, message)
+
+
+class PointSetNotSupported(NotImplementedError):
```

Comment:
In my mind, a `NotImplementedError` implies that this work is unfinished -- just needs to be implemented and is typically reserved for abstract methods. Most of the uses of `PointSetNotSupported` are for scenarios that are not possible and are outright not supported (e.g., dimension reducing operations) rather than things that have yet to be implemented. 

In my mind, more appropriate error types would be:

1. `TypeError`: there is an issue with the input type of the operation (`PointSet`)
2. `RuntimeError`: this is a general error for an unsupported operation

Reply:
Resolved in 08ee5943ed7213f4d119e12e6b127eb7d1447239.